modules/hdf5_handler/h5get.cc

```diff
@@ -524,8 +524,8 @@ void get_dataset(hid_t pid, const string &dname, DS_t * dt_inst_ptr)
         throw InternalErr(__FILE__, __LINE__, msg);
     }
 
-    hsize_t size[DODS_MAX_RANK];
-    hsize_t maxsize[DODS_MAX_RANK];
+    hsize_t size[ndims];
```

Comment:
Could these be vector<hsize_t> as above on lines 216?

Reply:
Oh, somehow those were skipped. changed them  and committed.